.github/workflows/ci.yml

```diff
@@ -1,14 +1,11 @@
-name: PR CI
+name: Business Filing UI CI
```

Comment:
FWIW, the repo is named Business **Filings** UI. If possible we should use the same labels everywhere.

Reply:
Done.